runtime/common/src/parachains.rs

```diff
@@ -550,8 +529,13 @@ fn majority_of(list_len: usize) -> usize {
 	list_len / 2 + list_len % 2
 }
 
-fn localized_payload<H: AsRef<[u8]>>(statement: Statement, parent_hash: H) -> Vec<u8> {
+fn localized_payload<S: AsRef<[u8]>, H: AsRef<[u8]>>(
+	statement: Statement,
+	session_index: S,
```

Comment:
```suggestion
	session_index: SessionIndex
```

Reply:
one step further (and even more neat IMO) is to have localized_payload accept `statement, signing_context` rather than force the caller to break it down externally.